### ISSUE TITLE
Select Multiple - Create Figma variant to support 'Create/Add' action 2

### ISSUE THREAD

**Issue by ashifalinadaf** [grommet/hpe-design-system#3507]
Working file -[ Figma](https://www.figma.com/file/EcLSFz5L7EwmOadmEYGshZ/SelectMultiple-Component?type=design&node-id=3607-18635&mode=design&t=snydAoLHzF7N6zTb-11)
Main ticket - https://github.com/grommet/hpe-design-system/issues/2887
Previous ticket- https://github.com/grommet/hpe-design-system/issues/3477
Guidance on when to use 'add', 'create', and 'new' - https://github.com/grommet/hpe-design-system/issues/3496
______________________________________________

- [x] Before modifying the existing figma component, create a plan and **specification** for the **modifications** that will be required to be made to the figma component.
-Document, ready it for review. [File ](https://www.figma.com/file/EcLSFz5L7EwmOadmEYGshZ/SelectMultiple-Component?type=design&node-id=3872-17703&mode=design&t=3qjLssWncOtzBihb-11)
-specifications and modifications for figma component. 
- [x] Identify **new properties or variance** that would need to be added to support this new feature. [File](https://www.figma.com/file/EcLSFz5L7EwmOadmEYGshZ/SelectMultiple-Component?type=design&node-id=3872-17703&mode=design&t=3qjLssWncOtzBihb-11)
-Properties 
-Variants
- [x] Verify the proposed specification is in alignment/**parity with the grommet** work that needs to happen
-Check with developers - [comments from developers](https://www.figma.com/file/EcLSFz5L7EwmOadmEYGshZ?node-id=3958:19176&mode=design#533903423)
- [x] **Confirm the placement and styling** of the new control in the SelectMultiple figma component
-Check with designers - [Design Proposal](https://www.figma.com/file/EcLSFz5L7EwmOadmEYGshZ/SelectMultiple-Component?type=design&node-id=3872-17703&mode=design&t=llV1ifxRbIMwxr9K-11)
- [ ] **Confirm the suggested interaction/behaviors** and containers to surface and render the new input field and save it
- [ ] Request reviews/feedback in-order to finalize the above

Deliverables: 
- [x] Completed design direction for "add item" flow
- [ ] **Create follow on issue** Create the **new SelectMultiple figma component** version supporting the new functionality
- [ ] **Create follow on issue for Grommet** to extend the SelectMultiple component to allow for a custom action button to be appended to the select's drop

**Comment by ashifalinadaf** [comment]
@KennyAtHPE @taysea @halocline 
Need some inputs for the changes [proposed for the Select Multiple component](https://www.figma.com/file/EcLSFz5L7EwmOadmEYGshZ/SelectMultiple-Component?type=design&node-id=3872-17703&mode=design&t=MgOicCF3Hq0K2jbn-11). 

@jcfilben Can you please have quick look at the [accessibility flows](https://www.figma.com/file/EcLSFz5L7EwmOadmEYGshZ/SelectMultiple-Component?type=design&node-id=3853-17645&mode=design&t=MgOicCF3Hq0K2jbn-11) and see if I am missing something before we connect on the same.

**Comment by jcfilben** [comment]
@ashifalinadaf added some comments in the Figma file [accessibility section](https://www.figma.com/file/EcLSFz5L7EwmOadmEYGshZ/SelectMultiple-Component?type=design&node-id=3853-17645&mode=design&t=MgOicCF3Hq0K2jbn-11)

**Comment by KennyAtHPE** [comment]
@ashifalinadaf left a few comments. I was a bit unclear on some of the visuals and suggestions. There were a couple of use cases that were mentioned but the flows didn't seem entirely fleshed out which was a bit hard to follow. When you're available, let's connect on this if possible.

On a side note, should this ticket be on the current sprint board instead of the other selectmultiple ticket?

**Comment by ashifalinadaf** [comment]
Hey @KennyAtHPE  Thanks for the feedback,
This needs to be in current sprint board, seems like you already moved it to current sprint board. 

Made [new flows](https://www.figma.com/file/EcLSFz5L7EwmOadmEYGshZ/SelectMultiple-Component?type=design&node-id=3958-19176&mode=design&t=ZLIAsSnWUE894GL9-11) easy to follow.

**Comment by ashifalinadaf** [comment]
Taylor has left [comments](https://www.figma.com/file/EcLSFz5L7EwmOadmEYGshZ?node-id=3958:19176&mode=design#533903423) on the [new flows](https://www.figma.com/file/EcLSFz5L7EwmOadmEYGshZ/SelectMultiple-Component?type=design&node-id=3958-19176&mode=design&t=ZLIAsSnWUE894GL9-11) file on exploring the option for both 'Add item' and Create 'typedinput' buttons. 
Need a check on the design from designers.

**Comment by ashifalinadaf** [comment]
After the discussion with designers during designers swarm, 
[Design ](https://www.figma.com/file/EcLSFz5L7EwmOadmEYGshZ/SelectMultiple-Component?type=design&node-id=3872-17703&mode=design&t=llV1ifxRbIMwxr9K-11) have been finalised to proceed with next step. 
Also the proposal for a new template to [input modal](https://www.figma.com/file/EcLSFz5L7EwmOadmEYGshZ/SelectMultiple-Component?type=design&node-id=3772-98843&mode=design&t=llV1ifxRbIMwxr9K-11). Created a ticket for the same. https://github.com/grommet/hpe-design-system/issues/3524

**Comment by SOjaHPE** [comment]
@ashifalinadaf  didn't see anything at this point that was concerning. I see you have the follow on ticket created, I'm comfortable w/ closing this one

**Comment by ashifalinadaf** [comment]
Closing the ticket, follow up ticket #3526
